content/quickstart/_index.md

```diff
@@ -200,26 +197,42 @@ For workers.dev domains, you will just need the Account ID:
 1. Log in to your Cloudflare account and select **Workers**.
 2. Scroll to the _API_ section and select **Click to copy** to copy your **Account ID**.
 
-### Global API Key
+## API Token
+
+1. Click **Get API Token** below the _API_ section to jump to your _Profile_ page.
+2. Click **Create Token** and select the **Start with Template** radio button. Select the **Edit Cloudflare Workers** template.
+3. Fill out the rest of the fields and then click **Continue to Summary**, where you can click **Create Token** and issue your token for use.
 
-1. Click **Get API Key** below the _API_ section to jump to your _Profile_ page.
+## Global API Key
+
+1. Click **Get API Token** below the _API_ section to jump to your _Profile_ page.
 2. Scroll to _API Keys_, and click **View** to copy your Global API Key **\***.
 
 **\* IMPORTANT: Treat your Global API Key like a password!**
 It should not be stored in version control or in your code, use environment variables if possible.
 
 ## Setup
 
-Set up your default credentials on your local machine via the `config` subcommand. You should only need to do this once. Running `wrangler config` will prompt you interactively for your email and API key:
+Set up your default credentials on your local machine via the `config` subcommand. This is an interactive command that will prompt you for your API token:
 
-```sh
-$ wrangler config
+```bash
+wrangler config
+Enter API token:
+superlongapitoken
+```
+
+You can also provide your email and global API key:
+
+```bash
+wrangler config --api-key
 Enter email:
-foo@bar.com
-Enter api key:
-123456abcdef
+testuser@example.com
+Enter global API key:
```

Comment:
Shouldn't this not be in the quickstart?

Reply:
this used to be in the quickstart, and i just updated it with the new config options. Whether we want to remove it or not should probably be handled in another PR (but fwiw I think it should stay)